processors/config/anaKalSimpTuple_cfg.py

```diff
@@ -9,6 +9,11 @@
                   help="Require positron in Top and Bottom", metavar="splitVolume", default=0)
 base.parser.add_argument("-R", "--region", type=str, dest="region",
                   help="Signal Region (SR) or Control Region (CR)", metavar="region", default="CR")
+base.parser.add_argument("-f", "--makeFlatTuple", type=int, dest="makeFlatTuple",
```

Comment:
I think we need to add this arguement to all cfg.py files that use a VertexAnaProcessor